docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md

```diff
@@ -133,11 +133,14 @@ For more information, read the
    ```
 
    For remote file systems you need to add the schema, it will be used to get the protocol being
-   used, for example:
+   used:
 
    ```toml
    [sources.filesystem] # use [sources.readers.credentials] for the "readers" source
-   bucket_url="s3://my-bucket/csv_files/"
+   # bucket_url="az://my-bucket/csv_files/" - for Azure Blob Storage
+   # bucket_url="gdrive://my-bucket/csv_files/" - for Google Drive folder
+   # bucket_url="gs://my-bucket/csv_files/" - for Google Storage
+   bucket_url="s3://my-bucket/csv_files/" # for AWS S3
```

Comment:
It'd be good to add every schema supported. There are examples in the doc, but not for all, so user, most likely, will have to verify what schema to set. But if they have a list of all supported schemas, they don't need to go anywhere else - the doc provides all.

Reply:
what do you mean by schema?

Reply:
ah, you talking about this 
![image](https://github.com/dlt-hub/dlt/assets/20367975/83d7044d-6463-4214-8e47-9634d8cc72b8)

Reply:
thank you! this is really helpful info